org.adempiere.base/src/org/compiere/model/X_AD_BroadcastMessage.java

```diff
@@ -47,6 +47,20 @@ public X_AD_BroadcastMessage (Properties ctx, int AD_BroadcastMessage_ID, String
         } */
     }
 
+    /** Standard Constructor */
+    public X_AD_BroadcastMessage (Properties ctx, int AD_BroadcastMessage_ID, String trxName, String ... virtualColumns)
+    {
+      super (ctx, AD_BroadcastMessage_ID, trxName, virtualColumns);
+      /** if (AD_BroadcastMessage_ID == 0)
+        {
+			setAD_BroadcastMessage_ID (0);
+			setBroadcastFrequency (null);
+			setBroadcastMessage (null);
+			setBroadcastType (null);
+			setTarget (null);
+        } */
+    }
+
```

Comment:
as it is, if your changes is for release_9 branch, you have to re-generate model using release_9 branch. The model class generated from master branch is not compatible with release_9 source.

Reply:
done

Reply:
> as it is, if your changes is for release_9 branch, you have to re-generate model using release_9 branch. The model class generated from master branch is not compatible with release_9 source.

This is going to do the maintenance of release-9 and master too difficult - I think better we are needing to integrate the changes on IDEMPIERE-5126